package/run.sh

```diff
@@ -19,13 +39,16 @@ else
     RESTART=false
 fi
 
-env "INSTALL_RKE2_ARTIFACT_PATH=${CATTLE_AGENT_EXECUTION_PWD}" installer.sh
+env "INSTALL_RKE2_ARTIFACT_PATH=${CATTLE_AGENT_EXECUTION_PWD}" INSTALL_RKE2_TAR_PREFIX="${SA_INSTALL_PREFIX}" installer.sh
+
+if [ ! -d "${SYSTEMD_BASE_PATH}" ]; then 
```

Comment:
I believe this is redundant as the `installer.sh` should create our `${SYSTEMD_BASE_PATH}`